sprouts.py

```diff
@@ -0,0 +1,246 @@
+import pygame
+from pygame import *
+import sys
+import time
+import random
+import math
+
+#Contains Game Settings: sproutset.py on Github
+from sproutset import Settings
+
+####OBJECTS - Will move later to .py or class
+#Game Window 
+pygame.init()
+ss = Settings()
+win = pygame.display.set_mode((ss.width, ss.height))
+pygame.display.set_caption('The Sprouts Game')
+
+running = True
+clock = pygame.time.Clock()
+
+#Player
+PlayerOne = 'Test1'
+PlayerOneScore = 0
+
+PlayerTwo = 'Test2'
+PlayerTwoScore = 0
+
+turn = 1 
+NumberStartingDots = 3
+
+#Dots
+dot_pos = []
+
+#Lines
+all_lines = []
+current_line = []
+
+####SPROUTS
+def sprouts():
+    global ss
+    #Start Screen
+    win.fill(ss.WHITE, None, 0)
+    text_writing('The Sprouts Game', ss.BLACK, 500, 25, 25) #Textwriting function
+    text_writing(f'{PlayerOne} score =' + str(PlayerOneScore), ss.BLACK, 125, 20, 12) #Updating Player Score text objects
+    text_writing(f'{PlayerTwo} score =' + str(PlayerTwoScore), ss.BLACK, 125, 40, 12)
+    pygame.draw.line(win, ss.BLACK, (0,52), (1000,52)) #Used to mimic an above playing field menu
+    pygame.display.update()
+
+    #Instructions screen 
+    #redraw_box()
+    #text_writing('The Rules:\n1. Not more than 3 lines per dot. Ect.', ss.BLACK, 500, 300, 25)    
```

Comment:
got a useful tipp from one of our peers here: we don't need to use \n for telling python to do a new line but can do it like this (example code from my project):
controlinfo = TextStim(win, text = '''Press the left arrow to defect 
					   and the right arrow to cooperate''',
					   font="Arial", height=20, color='black',
					    pos = (-300, -300))
so you can do ... text = ''' line
                                         .... line ....
                                         .... line .... 
                                         ... line ... '''

Reply:
Great tip!